draft-ecahc-moderation.md

```diff
@@ -310,6 +319,11 @@ decision by the IETF Chair can be appealed as described in
 {{!RFC2026}}, as with any other Area Director decision. In this case,
 the appeals chain starts with an appeal to the entire IESG.
 
+Moderators are not permitted to appeal working group chair decisions.
```

Comment:
Not sure on this.  I think that anyone should be able to appeal any decision and it should just take the normal escalation path.

Reply:
Ted Lemon and I also got into this issue on the list. This is brief:

https://mailarchive.ietf.org/arch/msg/mod-discuss/KCGz70SboXFXo3xc55E4QD2vbRE/

Reply:
Text removed